sound/soc/sof/intel/hda-dsp.c

```diff
@@ -776,7 +776,7 @@ int hda_dsp_resume(struct snd_sof_dev *sdev)
 		}
 
 		/* restore L1SEN bit */
-		if (hda->l1_support_changed)
+		if (hda->l1_disabled)
```

Comment:
Wouldn't it be simpler if this was l1_enabled, the name would match the mask field and functionality?

Reply:
@plbossart I thought about this but l1_enabled would make the logic more messy. Basically l1_enabled will be false by default during first boot and then we'd have to check if we need to enable it during first boot or not. With l1_disabled, we will always enable DMI L1 during first boot and then for every subsequent boot, it will depend on what is active

Reply:
I am not sure what this makes more complex. It's just messy as well when we have if (!hda->l1_disable), multiple negations give everyone a headache.

Reply:
I'd understand it if was multiple negations. But this is rather simple I think. My point is that my default l1 should be enabled and if I add that check, I'd have to set l1_enabled during probe. But with l1_disabled I dont need anything more